packages/cli/src/Doctor.ts

```diff
@@ -82,7 +82,8 @@ ${chalk.bold('Examples')}
     console.error(`👩‍⚕️🏥 Prisma Doctor checking the database...`)
 
     const connectionString = config.datasources[0].url
-    const canConnect = await canConnectToDatabase(connectionString.value, path.dirname(schemaPath))
+    // url.value exists because `ignoreEnvVarErrors: false` would have thrown an error if not
```

Comment:
Took me a bit to understand the comment. Maybe this way?
```suggestion
    // config.datasources[0].url.value exists because `ignoreEnvVarErrors: false` would have thrown an error if not
```